slidedeck/src/slides/03-react-state/00-react-state.pug

```diff
@@ -0,0 +1,107 @@
+section
+  h1 React and Local State
+
+section
+  h2 What is State?
+  ul
+    li <span class="component red">state</span> is your component's internally scoped ... state
+    li Used for the behavior of your component: open, disabled, submitted, etc.
+
+section
+  ul
+    li One exception: "Controller" components
```

Comment:
Is this new? I think one of the more confusing parts about React are all the terms around it. I'd like to limit the number of terms we use, but this one seems fine.

Reply:
Yeah I'm not either - this part was cut/pasted from another section. Seems like the standard terminology is Container component?